pkg/apis/numaflow/v1alpha1/pipeline_types.go

```diff
@@ -47,6 +47,11 @@ const (
 	// PipelineConditionDeployed has the status True when the Pipeline
 	// has its Vertices and Jobs created.
 	PipelineConditionDeployed ConditionType = "Deployed"
+
+	// PipelineConditionDaemonServiceHealthy has the status True when the daemon service is healthy.
+	PipelineConditionDaemonServiceHealthy    ConditionType = "DaemonServiceHealthy"
+	PipelineConditionSideInputServiceHealthy ConditionType = "SideInputManagerHealthy"
+	PipelineConditionVerticesServiceHealthy  ConditionType = "VerticesHealthy"
```

Comment:
To match the enum name

```suggestion
	PipelineConditionSideInputServiceHealthy ConditionType = "SideInputServiceHealthy"
	PipelineConditionVerticesServiceHealthy  ConditionType = "VerticesServiceHealthy"
```